app/js/main.js

```diff
@@ -203,6 +205,78 @@ function startFlowAuto() {
     });
     return;
   }
+
+  if (flow == 'metering') {
+    /* eslint-disable */
+
+    whenReady((subscriptions) => {
+      // Forget any subscriptions, for metering demo purposes.
+      subscriptions.reset();
+
+      // Set up metering demo controls.
+      MeteringDemo.setupControls();
+
+      // Example of a timestamp representing when a given action was taken.
+      const timestamp = 1597686771;
```

Comment:
nit: use current time.

Reply:
Initially I used `Date.now()` here, but John advised against it. He was worried developers would just copy `Date.now()` instead of providing a timestamp indicating when the event took place.